ntrans_dataprep.py

```diff
@@ -91,9 +107,24 @@ def generate_ngrams_from_corpus():
         root="BNC/Texts/", fileids=r"[A-K]/\w*/\w*\.xml"
     )
 
+    # Tells the program how to name the generated csv-files when splitting data into several
+    # files.
+    data_chunk = 1
+
     # To work with a sample size of the BNC, add a range in sents().
     # For example "for count, sentence in enumerate(bnc_corpus.sents()[:1000]):"
     for count, sentence in enumerate(bnc_corpus.sents()):
+        if count % 10000 == 0:
+            print(count)
+
+        # Splits up the proecssing of sentences into chunks of 300K, and clears the dict where
+        # the data is temporarily stored.
+        if count != 0 and count % 300000 == 0:
```

Comment:
Please assume that the person reading the code can actually read code. Don't explain it too much.

```suggestion
        if count != 0 and count % 300000 == 0:
```